roles/parse_operator_bundle/tasks/bundle_sanity_checks.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: "Include variables from annotations.yaml"
+  include_vars:
+    file: "{{ operator_work_dir }}/metadata/annotations.yaml"
+    name: annotations_vars
+
+- debug:
+    var: skopeo_inspect_json.Labels
+
+- debug:
+    var: annotations_vars.annotations
+
+- name: "Check if the annotation.yaml variables match the bundle image labels"
+  fail:
+    msg: "Values in the annotations yaml don't match the bundle image labels"
```

Comment:
Will ansible tell us which condition does not match? or we will see just debug message.
It would be good to have useful error reporting in case of failure.

Reply:
There are debug messages displaying the fields that will be tested prior to making the check, which should make it obvious which fields don't match. I'm open to splitting this into separate checks though.

Reply:
Sure they are there but it would be simpler to find a bug if we at least log "key" (e.g. `operators.operatorframework.io.bundle.channel.default.v1`) which is problematic/does not match

Reply:
Ok, changing